test/src/components/Button-test.js

```diff
@@ -26,4 +26,9 @@ describe('Button', () => {
     React.addons.TestUtils.Simulate.click(node);
     customHandleClick.called.should.be.true;
   });
+  it('should take in a color attribute', () => {
+    let coloredButton = render(<Button color={'teal'}>Submit</Button>).findTag('button');
+    coloredButton.props.className.should.contain('teal');
+    coloredButton.getDOMNode().getAttribute('class').should.contain('teal');
+  });
```

Comment:
No need to test `class` attribute of the DOM node if we test `className` contains `teal`.  It is enough to test our own code and prove `className` is set.  It is up to React's internals and tests to turn the `className` into DOM `class` attr on the DOM node.